### PR TITLE
Fix enrich command and trackdb status

### PR DESCRIPTION
**Pull request by bilalebi**

### CODE REVIEW

File: trackdbs/update_trackdb.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        print(f"{trackdbs_counter}/{total_trackdbs} trackdbs enriched/processed", end='\r')
+    print(f"{total_trackdbs} trackdbs enriched")
+
+
+def update_one_trackdb(trackdb_id):
+    """
+    Update the status of one specific trackdb and enrich ES docs
+    """
+    # Get the current trackdb object
+    one_trackdb = trackhubs.models.Trackdb.objects.filter(trackdb_id=trackdb_id).first()
```
Comment (bilalebi) [inline trackdbs/update_trackdb.py:42]: Handle non existent ids

File: trackdbs/update_trackdb.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    # Get all tracks belonging to the current trackdb object
+    all_tracks = trackhubs.models.Track.objects.filter(trackdb_id=trackdb_id)
+
+    if one_trackdb:
+        tracks_status = fetch_tracks_status(all_tracks, one_trackdb.source_url)
```
Comment (bilalebi) [inline trackdbs/update_trackdb.py:47]: Is there a way to ping just the header? using `urlib`

File: trackhubs/management/commands/enrich.py [hunk 3]
```diff
@@ -26,41 +29,63 @@
...
-        print("{} hubs couldn't be enriched".format(not_enriched_hubs_len))
+    def add_arguments(self, parser):
+        # This is mandatory argument
+        parser.add_argument(
+            'trackdbid',
+            type=str,
+            help="Update trackdb status by providing the <id> or 'all' \nif you want to update all trackdbs status",
+        )
+        # This is an optional argument
+        parser.add_argument(
```
Comment (bilalebi) [inline trackhubs/management/commands/enrich.py:66]: Remove exclude

File: trackhubs/tracks_status.py [hunk 4]
```diff
@@ -80,24 +83,24 @@ def fetch_tracks_status(trackdb_dict, trackdb_url):
...
-    for track in trackdb_dict:
-        if 'bigDataUrl' in track:
+    for track in tracks:
+        if track.big_data_url is not None:
             total_tracks_with_data += 1
             # get the full url for the bigDataUrl
-            big_data_full_url = fix_big_data_url(track['bigDataUrl'], trackdb_url)
+            big_data_full_url = fix_big_data_url(track.big_data_url, trackdb_url)
             # make sure it's working
             big_data_exists = check_response(big_data_full_url)
```
Comment (bilalebi) [inline trackhubs/tracks_status.py:92]: Is there a way to ping just the header? using `urlib`

**Review by sanjay-boddu — APPROVED** [review]

File: trackhubs/tracks_status.py (hunk 4 above)
Comment (bilalebi) [reply trackhubs/tracks_status.py:92]:
```
response = requests.head(url, allow_redirects=True)
```
take the same time as
```
response = urllib.request.urlopen(url)
```

**Pull request merged**